python/paddle/fluid/dygraph/dygraph_to_static/utils.py

```diff
@@ -471,11 +480,15 @@ def remove_if_exit(filepath):
     else:
         module = SourceFileLoader(module_name, f.name).load_module()
     func_name = dyfunc.__name__
-    if not hasattr(module, func_name):
+    if hasattr(module, '__impl__'):
```

Comment:
if we can't avoid check `__impl__`, I think we can change `__impl__` to a a more complex name such as `__i_m_p_l__`to avoid error handling for user-defined `__impl__` method

Reply:
Done,thx.

Reply:
Done,thx.